src/core/server/saved_objects/mappings/kibana_index_mappings_mixin.ts

```diff
@@ -17,51 +17,53 @@
  * under the License.
  */
 
+import { Server } from 'hapi';
```

Comment:
Using just the type `Server` from hapi.

Reply:
CI is failing on building types. I wonder, is there a version of the types that would work for hapi 14? https://libraries.io/npm/@types%2Fhapi/versions?page=2 Or should I type it as a hapi 17 server? Hm..